packages/shared/meta/selectors.ts

```diff
@@ -79,13 +79,13 @@ export function getFeatureFlags(store: RootMetaState): FeatureFlag {
     }
   }
   if (location.search.length !== 0) {
-    const flags = location.search.substr(1, location.search.length - 1).split('&')
-    flags.forEach((element) => {
-      if (element.includes(`DISABLE_`)) {
-        const featureName = element.replace('DISABLE_', '').toLowerCase()
+    const flags = new URLSearchParams(location.search)
+    flags.forEach((_, key) => {
+      if (key.includes(`DISABLE_`)) {
+        const featureName = key.replace('DISABLE_', '').toLowerCase()
         featureFlag.flags[featureName] = false
-      } else if (location.search.includes(`ENABLE_`)) {
```

Comment:
This could be causing other problems.
If you use `ENABLE_x` you were adding a feature to the feature map for every parameter that you had in the url.

So for example with the following URLs:
```
&ENABLE_x&hello&how&are&you
```
the featureFlag.flags dictionary would be:

```
flags['x'] = true
flags['hello'] = true
flags['how'] = true
flags['are'] = true
flags['you'] = true
```